nsls2forge_utils/dashboard.py

```diff
@@ -40,16 +69,19 @@ def create_dashboard(names=None, write_to='README.md'):
       defaults_version='[![defaults version](https://img.shields.io/conda/vn/anaconda/{name})]'
                        '(https://anaconda.org/anaconda/{name})',
       pypi_version='[![PyPI version](https://img.shields.io/pypi/v/{name})](https://pypi.org/project/{name}/)',
+      github_version='[![GitHub version](https://img.shields.io/github/v/tag/{org}/{name})]'
```

Comment:
Are we sure the `name` always matches the name of the conda package? There are cases when it can differ.

Reply:
The name comes from the feedstock name, can I see an example where this happens?

It should be an easy change, I would change `name` to something else and then also extract the real repo name from the `home:` or `dev_url:` and use that instead.

Reply:
Here is an [example](https://github.com/nsls-ii-forge/arpack-mpi-feedstock/blob/270dae599693bc7b7407c80de82cc242074fef13/recipe/meta.yaml#L34) of such a mismatch:
```yaml
  home: https://github.com/opencollab/arpack-ng
```
while the package is named `arpack-mpi`.

It's not mandatory to call the package the same way. For example, there is another confusion between the same package on PyPI (`PyQt5`) and conda (`pyqt`).

I think we should attempt to use the extracted repo name from the URL, and if it's not available (like in the case of HXN), try to use the package name.